Author: Reiner Herrmann <reiner@reiner-h.de>
Description: Sort list of source files for deterministic linking order

--- a/lib/Makefile
+++ b/lib/Makefile
@@ -4,7 +4,7 @@
 RELEASE_CXXFLAGS=-O2 -Wall -Wextra
 FUSEFLAGS=$(shell pkg-config fuse --cflags)
 ZIPFLAGS=$(shell pkg-config libzip --cflags)
-SOURCES=$(wildcard *.cpp)
+SOURCES=$(sort $(wildcard *.cpp))
 OBJECTS=$(SOURCES:.cpp=.o)
 CLEANFILES=$(OBJECTS) $(DEST)
 
